File: src/main/java/com/dsm/pyeongan/domains/service/DiaryBookRequestConnectionService.java
package com.dsm.pyeongan.domains.service;

import retrofit2.Call;
import retrofit2.http.*;

public interface DiaryBookRequestConnectionService {
    @Headers(value = {"accept: application/json", "content-type: application/json"})
    @POST("/repositories/diary-book")
    Call<String> addRequest(@Header("userId") String userId, @Body String code);
}

File: src/main/java/com/dsm/pyeongan/domains/service/DiaryBookRequestConnectionService.kt
package com.dsm.pyeongan.domains.service

import retrofit2.Call
import retrofit2.http.*

interface DiaryBookRequestConnectionService {
    @Headers("accept: application/json", "content-type: application/json")
    @GET("/repositories/diary-book")
    fun addRequest(@Header("id") id: String, @Body code: String): Call<Unit>
}